### PR TITLE
Add spread 2 webpages

### PR DESCRIPTION
**Pull request by aesha-spreadd**
Add 2 given spread pages
Screenshot 2024-01-05 at 12 59 01 PM

Screenshot 2024-01-04 at 6 37 03 PM

- [x] Checking the code is duplicate or not?
- [x]  Is these code has bug
- [ ]  Is code is check by shubhum sir or ankit sir or tara ma'am?
- [ ]  Are you going to merge this or not?

### CODE REVIEW

File: src/Components/CreatorStore.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+              </ImageWrapper>
+              <PopupWrapper>
+                <h2>100+ curated products</h2>
+              </PopupWrapper>
+            </CreatorStyledLeftInner>
+            <RightInnerWrapper>
+              <RightWrapperContent>
+                <CreatorsStore>Creator Store</CreatorsStore>
+                <StyledText>
+                  <StyledTitle>Curate your</StyledTitle>
```
Comment (ankitadrixus08) [inline src/Components/CreatorStore.js]: make this all title in one line and set with width @aeshapatel025

File: src/Components/CreatorStore.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                <p>Happy Customers</p>
+              </CreatorStyledRateBox>
+              <ImageWrapper>
+                <img
+                  src={'https://spreadd.io/assets/home/iphone.png'}
+                  alt="iPhone"
+                />
+              </ImageWrapper>
+              <PopupWrapper>
+                <h2>100+ curated products</h2>
```
Comment (ankitadrixus08) [inline src/Components/CreatorStore.js]: need class here don't put css on tag if not required @aeshapatel025

File: src/Components/CreatorStore.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+              <RightWrapperContent>
+                <CreatorsStore>Creator Store</CreatorsStore>
+                <StyledText>
+                  <StyledTitle>Curate your</StyledTitle>
+                  <StyledTitle>products and turn</StyledTitle>
+                  <StyledTitle>your content shop-</StyledTitle>
+                  <StyledTitle>able</StyledTitle>
+                </StyledText>
+                <StyledParagraph>
+                  <StyledSpanText>
```
Comment (ankitadrixus08) [inline src/Components/CreatorStore.js]: same as above make that in one line code and set with width attribute @aeshapatel025

File: src/Components/CreatorStore.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  RightWrapperContent,
+  StyledSpanText,
+  StyledParagraph,
+  GlobalStyles,
+} from './CreatorsStoreStyled';
+
+const CreatorStore = () => {
+  return (
+    <div>
+      <GlobalStyles />
```
Comment (ankitadrixus08) [inline src/Components/CreatorStore.js]: what's the use of this @aeshapatel025

File: src/Components/CreatorsStoreStyled.js [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+  @font-face {
+    font-family: 'ProximaNova';
+    src: url('${ProximaNova}') format('truetype');
+    font-style: normal;
+  }
+`;
+
+export const CampaignMarketSection = styled.section`
+  margin: 0 auto;
+  background: rgb(255, 246, 246);
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js]: use hexcode instead of this @aeshapatel025

File: src/Components/CreatorsStoreStyled.js [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+
+export const RightInnerWrapper = styled.div`
+  margin: 0 0 0 30px;
+  max-width: 48%;
+  width: 100%;
+  display: flex;
+  font-family: 'ProximaNova', sans-serif;
+
+  @media (max-width: 766px) {
+    order: -1;
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js]: why use this @aeshapatel025

File: src/Components/CreatorsStoreStyled.js [hunk 7]
```diff
@@ -0,0 +1,259 @@
...
+  font-family: 'ProximaNova', sans-serif;
+`;
+
+export const CreatorStyledRateBox = styled.div`
+  position: absolute;
+  right: 112px;
+  top: -13px;
+  padding: 10px;
+  font-family: 'ProximaNova', sans-serif;
+  background: white;
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js]: use code here @aeshapatel025

File: src/Components/CreatorsStoreStyled.js [hunk 8]
```diff
@@ -0,0 +1,259 @@
...
+  box-shadow: rgba(240, 97, 86, 0.1) 5px 12px 21px;
+
+  h5 {
+    font-size: 14px;
+    font-family: 'ProximaNova', sans-serif;
+    line-height: 14px;
+    color: rgb(255, 78, 83);
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js]: use hex code here @aeshapatel025

File: src/Components/CreatorsStoreStyled.js [hunk 9]
```diff
@@ -0,0 +1,259 @@
...
+    margin: 2px;
+    font-weight: bold;
+  }
+
+  p {
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js]: dont give css on tag @aeshapatel025

File: src/Components/CreatorsStoreStyled.js [hunk 10]
```diff
@@ -0,0 +1,259 @@
...
+    font-size: 14px;
+    line-height: 22px;
+  }
+
+  @media (max-width: 991px) {
+    right: 120px;
+    top: 23px;
+    padding: 6px 6px;
+
+    h5 {
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js]: same as above @aeshapatel025

File: src/Components/CreatorsStoreStyled.js [hunk 11]
```diff
@@ -0,0 +1,259 @@
...
+      font-size: 12px;
+    }
+
+    p {
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js]: same as above @aeshapatel025

File: src/Components/CreatorsStoreStyled.js [hunk 12]
```diff
@@ -0,0 +1,259 @@
...
+      font-size: 10px;
+    }
+  }
+
+  @media (max-width: 720px) {
+    right: -4px;
+    top: 23px;
+    padding: 6px 6px;
+
+    h5 {
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js]: same as above @aeshapatel025

File: src/Components/CreatorsStoreStyled.js [hunk 13]
```diff
@@ -0,0 +1,259 @@
...
+  img {
+    max-width: 398px;
+    width: 100%;
+    top: -220px;
+  }
+
+  @media (max-width: 991px) {
+    margin: 40px 0 0 10px;
+    img {
+      max-width: 298px;
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js:141]: always use width with max-width @aeshapatel025

File: src/Components/CreatorsStoreStyled.js [hunk 14]
```diff
@@ -0,0 +1,259 @@
...
+    background: linear-gradient(
+      240deg,
+      rgb(240, 97, 86) 70%,
+      rgba(240, 97, 86, 0.4) 100%
+    );
+    left: -1425px;
+    transform: translateX(-55%);
+    padding: 20px;
+    opacity: 0.8;
+    width: 4250px;
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js]: wrong width here @aeshapatel025

File: src/Components/CreatorsStoreStyled.js [hunk 15]
```diff
@@ -0,0 +1,259 @@
...
+    height: 60px;
+    display: flex;
+    top: 285px;
+    align-items: center;
+  }
+
+  h2 {
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js]: dont use css on tag @aeshapatel025 give class for that

File: src/Components/SpreadAcademeyStyled.js [hunk 16]
```diff
@@ -0,0 +1,353 @@
...
+    }
+  }
+`;
+
+export const SpreadAcademyQuote = styled.div`
+  position: absolute;
+  top: -25px;
+
+  transform: translateX(50%);
+  background: rgb(254, 79, 82);
```
Comment (ankitadrixus08) [inline src/Components/SpreadAcademeyStyled.js]: @aeshapatel025 USE hexcode here

File: src/Components/SpreadAcademy.js [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+            <SpreadRightInner>
+              <SpreadRightDetails>
+                <SpreadAcadameyTag>Spreadd Academy</SpreadAcadameyTag>
+                <StyledText>
+                  <StyledTitle>Create courses and</StyledTitle>
+                  <StyledTitle>curriculums for your</StyledTitle>
+                  <StyledTitle>collaborators/followers</StyledTitle>
+                </StyledText>
+                <StyledParagraph>
+                  <StyledParagraphText>
```
Comment (ankitadrixus08) [inline src/Components/SpreadAcademy.js]: use all 3 text in one line @aeshapatel025

**Review by ankitadrixus08 — COMMENTED** [review]
17m

File: src/Components/SpreadAcademy.js (hunk 17 above)
Comment (aesha-spreadd) [reply src/Components/SpreadAcademy.js]: changes done

File: src/Components/CreatorStore.js (hunk 1 above)
Comment (aesha-spreadd) [reply src/Components/CreatorStore.js]: changes done

File: src/Components/CreatorsStoreStyled.js (hunk 13 above)
Comment (aesha-spreadd) [reply src/Components/CreatorsStoreStyled.js:141]: changes done

File: src/Components/CreatorsStoreStyled.js [hunk 18]
```diff
@@ -0,0 +1,262 @@
...
+export const PopupWrapper = styled.div`
+  &::after {
+    position: absolute;
+    content: '';
+    background: linear-gradient(240deg, #ff4e53 70%, #f06156 100%);
+    left: -650px;
+    transform: translateX(-55%);
+    padding: 20px;
+    opacity: 0.8;
+    width: 2550px;
```
Comment (ankitadrixus08) [inline src/Components/CreatorsStoreStyled.js:176]: why 2550?? @aeshapatel025

File: src/Components/CreatorStore.js [hunk 19]
```diff
@@ -0,0 +1,74 @@
...
+            <RightInnerWrapper>
+              <RightWrapperContent>
+                <CreatorsStore>Creator Store</CreatorsStore>
+                <StyledText>
+                  <div>
+                    Curate your products and turn your content shop- able
+                  </div>
+                </StyledText>
+                <StyledParagraph>
+                  <div>
```
Comment (ankitadrixus08) [inline src/Components/CreatorStore.js]: use p tag here @aeshapatel025

File: src/Components/SpreadAcademy.js [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+              <SpreadRightDetails>
+                <SpreadAcadameyTag>Spreadd Academy</SpreadAcadameyTag>
+                <StyledText>
+                  <div>
+                    Create courses and curriculums for your
+                    collaborators/followers
+                  </div>
+                </StyledText>
+                <StyledParagraph>
+                  <div>
```
Comment (ankitadrixus08) [inline src/Components/SpreadAcademy.js]: same use p tag here @aeshapatel025

**Review by ankitadrixus08 — APPROVED** [review]
5m

File: src/Components/CreatorsStoreStyled.js (hunk 18 above)
Comment (aesha-spreadd) [reply src/Components/CreatorsStoreStyled.js:176]: It is a part of that pink line of spread page that`s why i have given 2550px width to it

**Comment by aesha-spreadd** [comment]
Add spread two page task

**Comment by aesha-spreadd** [comment]
Add Spread two page of task

**Pull request merged**